pkg/vmimage/data/script.sh

```diff
@@ -42,8 +42,9 @@ cat >client-key.pem <<'EOF'
 {{ .Builder.ClientKey | PrivateKeyAsBytes | String }}
 EOF
 
-go get github.com/jim-minter/tlsproxy
-go/bin/tlsproxy -insecure -key client-key.pem -cert client-cert.pem https://cdn.redhat.com/ &
+go get github.com/openshift/openshift-azure/cmd/azure
+set +x
+go/bin/azure tlsproxy -insecure -key client-key.pem -cert client-cert.pem https://cdn.redhat.com/ &
```

Comment:
vmimage test now failing on this line, I don't think this is the right binary... was hoping I could just rebase and merge but looks like this will have to wait until next image publication :(